e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -125,6 +126,21 @@ Scenario('HMCTS admin enters hearing details and submits', async (I, caseViewPag
   I.seeAnswerInTab(9, 'Hearing 2', 'Judge or magistrate\'s last name', hearingDetails[1].lastName);
 });
 
+Scenario('HMCTS admin creates C21 order for the case', async (I, caseViewPage, createC21OrderEventPage) => {
+  await caseViewPage.goToNewActions(config.administrationActions.createC21Order);
+  await createC21OrderEventPage.enterOrder();
+  await I.click('Continue');
+  await createC21OrderEventPage.enterJudgeAndLegalAdvisor('Sotomayer', 'Peter Parker');
+  await I.completeEvent('Save and continue');
+  const now = new Date();
+  I.seeEventSubmissionConfirmation(config.administrationActions.createC21Order);
+  caseViewPage.selectTab(caseViewPage.tabs.orders);
+  I.seeAnswerInTab(1, 'C21 Order 1', 'File name', 'C21_Order_1.pdf');
+  I.seeAnswerInTab(2, 'C21 Order 1', 'Order title', 'Example Title');
+  I.seeAnswerInTab(3, 'C21 Order 1', 'Date and time of upload', dateFormat(now, 'd mmmm yyyy'));
```

Comment:
hmm, if I run this test at midnight, I will probably encounter times where the time of upload and the time of now is different 🤔

Reply:
yes this is a possibility - do you have any suggestions for handling that? initially I didn't test the date at all as I didn't know how to deal with that, adam asked me to try something so me and @lewisbirks made this as our best attempt

Reply:
hmm dates are hard, I guess this is fine to leave for now 👍